guidelines/enhancement_template.md

```diff
@@ -72,17 +72,33 @@ this proposal. Describe why the change is important and the benefits to users.
 
 ### User Stories
 
-Detail the things that people will be able to do if this is
-implemented. In each story, explain who the actor is based on their
-role, explain what they want to do with the system, and explain the
-underlying goal they have. Use the standard three part formula:
+Detail the things that people will be able to do if this is implemented and
+what goal that allows them to achieve .In each story, explain who the actor
+is based on their role, explain what they want to do with the system,
+and explain the underlying goal they have, what it is they are going to
+achieve with this new feature.
 
-"As a <role>, I want to <take some action> so that I can <accomplish a
-goal>."
+Use the standard three part formula:
+
+"As a \<role\>, I want to \<take some action\> so that I can \<accomplish a
+goal\>."
 
 Make the change feel real for users, without getting bogged down in
 implementation details.
 
+Here are some example user stories to show what they might look like:
+
+* As an OpenShift engineer, I want to write an enhancement, so that I can share my feature idea
+and get feedback from others.
+* As a product manager, I want the engineers to implement this RFE, so that my customer remains happy.
```

Comment:
i think this example will lead people into producing bad user stories, as it does not really tie together why "implementing this rfe" will lead to the outcome of "my customer remains happy".

it's pretty close to the anti-pattern of "i want to implement this thing, so that this thing will be implemented"

Reply:
I agree.

"As a product manager, I want to review this enhancement proposal, so that I can make sure the customer requirements are met by the design."